public/static/images/twitter-icon-blue.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="14" viewBox="0 0 16 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Can we rather use something like `import { Twitter } from '@styled-icons/fa-brands/Twitter';`?

Reply:
I've changed that. Thanks for pointing it out. 😄